src/System.Management.Automation/engine/Credential.cs

```diff
@@ -273,9 +273,10 @@ public NetworkCredential GetNetworkCredential()
                 if (IsValidUserName(_userName, out user, out domain))
                 {
 #if CORECLR
-                    // NetworkCredential constructor only accepts plain string password in CoreCLR
-                    // Since user can already access the plain text password via PSCredential.GetNetworkCredential().Password,
-                    // this change won't be a security issue for PS on CSS.
+                    // NetworkCredential constructor only accepts plain string password in .NET Core.
```

Comment:
Please add a "TODO: " to this comment so that we don't forget about this work item.